operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,432 @@
+# iSCSI SBPS (Scalable Boot Content Projection Service)
+
+* [Introduction](#introduction)
+* [iSCSI SBPS solution details](#iscsi-sbps-solution-details)
+* [Steps to achieve SBPS](#steps-to-achieve-sbps)
+* [Steps to switch from DVS to iSCSI](#steps-to-switch-from-dvs-to-iscsi)
+* [Glossary](#glossary)
+
+## Introduction
+
+iSCSI based boot content projection solution named Scalable Boot Content Projection Service (SBPS)
+is an alternate boot content projection solution and planning to replace current DVS (Cray Data Virtualization Service)
+to project boot content like `rootfs` and `programming environment` (PE) images. SBPS is aimed to offer better
+reliability, availability, security, ease and speed of deployment and ease of management than DVS.
+SBPS solution is offered from CSM 1.6 onwards and this solution is spread across different components BOS, USS/COS
+and the core service SBPS Marshal agent is delivered as an RPM gets deployed as part of CFS `ansible` play books.
+In CSM 1.6, both DVS and SBPS will co-exist and DVS is planned to deprecate in CSM 1.7
+
+**SBPS Key features:**
+
+* Provides open source friendly solution for read-only content projection (`rootfs` and `PE`) as it uses LIO (Linux IO) which is open source.
+* Horizontally scalable content projection service (iSCSI target side)
+* Delivers active/active IO operation from iSCSI initiator(s) to content projection service
+* Delivers seamless failover and failback for iSCSI initiator(s) on iSCSI target(s) or partial network failure
+* Supports projection over HSN (High Speed Network) and NMN (Node Management Network)
+* Does not require additional hardware infrastructure (iSCSI target)
+* Co-exists with DVS until it is deprecated (Plan to deprecate DVS in CSM 1.7)
```

Comment:
Maybe say `(Plan to deprecate DVS for rootfs projection in CSM 1.7)` as DVS is used by customers for purposes other than rootfs projection using non management nodes.   That use will continue to be supported.

Reply:
ok sure

Reply:
```suggestion
* Co-exists with DVS until it is deprecated (Plan to deprecate DVS for rootfs projection in CSM 1.7)
```